packages/web-app-admin-settings/src/index.ts

```diff
@@ -21,12 +22,21 @@ const routes = [
   {
     path: '/',
     redirect: () => {
-      if (permissionManager().hasUserManagement()) {
-        return { name: 'admin-settings-users' }
+      if (permissionManager().hasSystemManagement()) {
+        return { name: 'admin-settings-general' }
```

Comment:
I know the condition for `hasUserManagement` and `hasSystemManagement` are identical currently, but how's that in the future? Because users without permissions for `systemManagement`, but for `userManagement`, will now be directed to `admin-settings-spaces`.

Reply:
I don't know about the future plans an the RBAC system. 
Currently, this is just an role=admin check.